src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int SIZE = 12;
+    public static final int EMPTY = -1;
+    public static final int FULL = 11;
```

Comment:
This constant is related to SIZE (yet, the name is misleading). Changing only one leads to an error (inconsistency).